test/extended/util/db/mysql.go

```diff
@@ -80,6 +81,11 @@ func (m MySQL) QueryPrivileged(oc *util.CLI, query string) (string, error) {
 			query, masterConf.Env["MYSQL_DATABASE"])).Output()
 }
 
+// QueryPrimary queries the database on primary node as a regular user.
+func (m MySQL) QueryPrimary(oc *util.CLI, query string) (string, error) {
+	return "", errors.New("not implemented")
```

Comment:
I don't mind. Actually, there was another interface `ReplicaSet` with this method but later I decided to join them together.